Vizzuality/marxan-cloud
# [MRXN23-237] (feat): scenario list

## New Project panel: scenario list

### Overview

New layout for project panel scenario list

### Designs
[Scenario list view prototype with no scenarios](https://www.figma.com/proto/0xD4gR9pCqU7NROnlSW6ck/Marxan-Layout---Redesign?node-id=568-40437&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=568%3A40417&show-proto-sidebar=1)
[Scenario list view prototype with some scenarios](https://www.figma.com/proto/0xD4gR9pCqU7NROnlSW6ck/Marxan-Layout---Redesign?node-id=521-57939&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=521%3A58646)

### Testing instructions
- Create a project and check you can edit name and description, publish and unpublish it and download. Also try to add someone to this project.
- Create a project and check if you are able to create one scenario or navigate to upload features, protected areas or cost surface tabs.
- Once you have some scenarios, check if you can see the settings, duplicate and download each scenario and also see its detail. 

### Feature relevant tickets

[MRXN23-237](https://vizzuality.atlassian.net/browse/MRXN23-237)



[MRXN23-237]: https://vizzuality.atlassian.net/browse/MRXN23-237?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
